uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/documents.js

```diff
@@ -54,9 +54,14 @@ module.exports = page.extend({
   },
 
   /**
-   * Waits until the specified select is visible on the page
+   * Waits until the user name in the "Staff" column and the filter dates are visible
    */
   async waitForReady () {
     await this.puppet.waitFor('.ct-filter-date', { visible: true });
+    await this.puppet.waitFor('.ct-table-documents [href^="/civicrm/contact/view"]', { visible: true });
```

Comment:
I was checking the UI, this selector is present twice. So It might be due to one of the elements being visible before the other. Could you check by waiting for individual items like
`.ct-table-documents td:nth-child(2) [href^="/civicrm/contact/view"` and
`.ct-table-documents td:nth-child(3) [href^="/civicrm/contact/view"`

Reply:
it's not important if there is one or multiple selectors (it really depends on the data in your local site), as long as one of element that matches that selector is visible, then backstop gets the green light